ci/scripts/e2e-source-test.sh

```diff
@@ -32,6 +32,7 @@ echo "--- Download mise"
 buildkite-agent artifact download avro-simple-schema.avsc ./
 buildkite-agent artifact download avro-complex-schema.avsc ./
 buildkite-agent artifact download proto-complex-schema ./
+buildkite-agent artifact download proto-metabit-assembly-schema ./
```

Comment:
These 4 files are uploaded in the build stage but only used in e2e-source-test stage. Why not directly use the one in the repo / do local copy?